WaterLog_Backend/Models/DataPoints.cs

```diff
@@ -52,6 +52,17 @@ public List<Y> getv()
             }
             return outV;
         }
+
+        public List<T> getv2()
```

Comment:
What is v?
The method name can be better

Reply:
Its template variables, this class can any type of variabel combination <T,V>

Reply:
we use it for all formats eg. Datapoint<int,double>,Datapoint<String,double> etc

Reply:
but getV2 is not good at all

Reply:
all chnages done